.github/workflows/misspell-fixer.yml

```diff
@@ -0,0 +1,16 @@
+name: Misspell fixer
```

Comment:
Just a general question: Does this spell checker action guarantees that compilation will not fail? As I see it renames function names as well. Or does it only corrects spelling inside quotes?

Reply:
I don't know, but if it fails, we can revert it or fix it after merge.